diff -ur lxpanel-0.10.0.old/plugins/launchtaskbar.c lxpanel-0.10.0/plugins/launchtaskbar.c
--- lxpanel-0.10.0.old/plugins/launchtaskbar.c	2019-11-08 13:47:44.468975969 +0400
+++ lxpanel-0.10.0/plugins/launchtaskbar.c	2019-11-08 13:49:46.771871512 +0400
@@ -1121,9 +1121,7 @@
                                    special_cases_filepath,
                                    G_KEY_FILE_KEEP_COMMENTS, NULL))
     {
-        launchtaskbar_constructor_add_default_special_case(ltbp, "synaptic", "synaptic-pkexec");
-        launchtaskbar_constructor_add_default_special_case(ltbp, "soffice.bin", "libreoffice");
-        launchtaskbar_constructor_add_default_special_case(ltbp, "x-terminal-emulator", "lxterminal");
+        launchtaskbar_constructor_add_default_special_case(ltbp, "x-terminal-emulator", "xterm");
         gchar *key_file_data = g_key_file_to_data(ltbp->p_key_file_special_cases, NULL, NULL);
         g_file_set_contents(special_cases_filepath, key_file_data, -1, NULL);
         g_free(key_file_data);
diff -ur lxpanel-0.10.0.old/src/misc.c lxpanel-0.10.0/src/misc.c
--- lxpanel-0.10.0.old/src/misc.c	2019-11-08 13:47:44.468975969 +0400
+++ lxpanel-0.10.0/src/misc.c	2019-11-08 13:49:24.648617215 +0400
@@ -1633,7 +1633,7 @@
     {
 	char * escaped_cmd = g_shell_quote(cmd);
         char* term_cmd;
-        const char* term = fm_config->terminal ? fm_config->terminal : "lxterminal";
+        const char* term = fm_config->terminal ? fm_config->terminal : "xterm";
         if( strstr(term, "%s") )
             term_cmd = g_strdup_printf(term, escaped_cmd);
         else
