diff --git a/client/dhclient.conf.example b/client/dhclient.conf.example
index 2c35d24..8de48dd 100644
--- a/client/dhclient.conf.example
+++ b/client/dhclient.conf.example
@@ -1,3 +1,7 @@
+#
+# /etc/dhclient.conf: ISC dhclient(8) configuration file
+#
+
 send host-name = pick-first-value(gethostname(), "ISC-dhclient");
 send dhcp-client-identifier 1:0:a0:24:ab:fb:9c;
 send dhcp-lease-time 3600;
@@ -11,7 +15,7 @@ retry 60;
 reboot 10;
 select-timeout 5;
 initial-interval 2;
-script "/etc/dhclient-script";
+script "/sbin/dhclient-script";
 media "-link0 -link1 -link2", "link0 link1";
 reject 192.33.137.209;
 
diff --git a/server/dhcpd.conf.example b/server/dhcpd.conf.example
index 8b99f49..2ca1bc2 100644
--- a/server/dhcpd.conf.example
+++ b/server/dhcpd.conf.example
@@ -1,11 +1,11 @@
-# dhcpd.conf
 #
-# Sample configuration file for ISC dhcpd
+# /etc/dhcpd.conf: ISC dhcpd(8) configuration file
 #
 
-# option definitions common to all supported networks...
+# Option definitions common to all supported networks.
 option domain-name "example.org";
 option domain-name-servers ns1.example.org, ns2.example.org;
+ddns-update-style ad-hoc;
 
 default-lease-time 600;
 max-lease-time 7200;
